builders/substrate/interfaces/features/governance/parameters.md

```diff
@@ -0,0 +1,80 @@
+---
+title: Parameters Pallet
+description: Learn how Moonbeam's Parameters Pallet safely and dynamically modifies network config items via on-chain governance, removing the need for runtime upgrades.
+keywords: democracy, substrate, pallet, moonbeam, polkadot, voting, vote, referenda, parameters
+---
+
+# The Parameters Pallet
+
+## Introduction {: #introduction }
+
+The Parameters Pallet on Moonbeam is a governance-focused module that enables community-driven modifications to key network configuration items, such as the deposit requirement for the randomness module, directly on-chain through proposals and votes, without necessarily requiring a runtime upgrade. By removing the need for frequent code changes, the Parameters Pallet allows the network to respond more quickly to evolving needs while maintaining transparency and consensus.
+
+This guide will provide an overview of the extrinsics, storage methods, and getters for the pallet constants available in the Parameters Pallet on Moonbeam. This guide assumes you are familiar with governance-related terminology; if not, please check out the [governance overview page](/learn/features/governance/#opengov){target=_blank} for more information.
+
+## Parameters Pallet Interface {: #parameter-pallet-interface }
+
+### Extrinsics {: #extrinsics }
+
+The Parameters Pallet provides one extrinsic (functions):
+
+???+ function "**setParameter**(keyValue) - Sets the value of a parameter"
+
+    === "Parameters"
+
+        - `keyValue` - The key to the storage item to be modified. 
+
+    === "Example"
+
+        Suppose you want to adjust the deposit parameter for the randomness pallet. You'd start by crafting a call to the `setParameter` function, specifying the randomness pallet's key and the new deposit value. In Polkadot.js Apps, this involves selecting `parameters.setParameter(keyValue)` and then updating the Deposit field for `PalletRandomness`. While you can generate and review this call data beforehand, the actual change must still go through the governance process—meaning it needs to be proposed, voted on, and approved by the community before the new deposit value takes effect on-chain.
```

Comment:
Is "Deposit" capitalized for a reason?

This is an interesting take on an example. I'm not sure about this 🤔